<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserAdmin</class>
 <widget class="QDialog" name="UserAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="CreateTab_2">
       <attribute name="title">
        <string>Create</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>420</y>
          <width>491</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Status:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ustatusLabel2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>491</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="4,6">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>X Coordinate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="xInput_2"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_19" stretch="4,6">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Y Coordinate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="yInput_2"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="4,6">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Radius</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="radiusInput"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_20">
           <item>
            <widget class="QPushButton" name="addBtnManual_2">
             <property name="text">
              <string>Add user</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearBtnCreate_2">
             <property name="text">
              <string>Clear values</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>310</y>
          <width>489</width>
          <height>50</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="4,4">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Power</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="assignedPowLbl">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>20</y>
          <width>251</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Make a request to access White Space:</string>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>491</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Closest transmitter:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="tmitDistLbl">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="tmitDistLbl_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>230</y>
          <width>118</width>
          <height>3</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="Line" name="line_3">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>390</y>
          <width>118</width>
          <height>3</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="ModifyTab_2">
       <attribute name="title">
        <string>Modify</string>
       </attribute>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>17</x>
          <y>80</y>
          <width>491</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>371</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_21">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Choose a user to edit:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="userDropDown">
           <property name="currentText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>491</width>
          <height>381</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="modifyLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_22" stretch="4,6">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Power</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="powerInputModify_2"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_23" stretch="4,6">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Radius</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="radiusInputModify"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_24" stretch="4,6">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>X Coordinate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="xInputModify_2"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_25" stretch="4,6">
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Y Coordinate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="yInputModify_2"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_26">
           <item>
            <widget class="QPushButton" name="ModifyBtn_2">
             <property name="text">
              <string>Apply user changes</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Status:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ustatusLabel3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Closest transmitter:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="tmitDistLbl_3">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="tmitDistLbl_4">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_28" stretch="4,4">
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Power</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="assignedPowLbl_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="RemoveTab_2">
       <attribute name="title">
        <string>Remove</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_8">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>350</y>
          <width>401</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="removeBtn">
             <property name="text">
              <string>Remove Records</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clrSelectBtn">
             <property name="text">
              <string>Clear Selections</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_27">
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Status:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ustatusLabel5">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QTableView" name="uListView_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>501</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="ListTab_2">
       <attribute name="title">
        <string>List</string>
       </attribute>
       <widget class="QTableView" name="uListView">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>11</y>
          <width>511</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>400</y>
          <width>511</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="uExportBtn">
             <property name="text">
              <string>Export to File</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Status:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ustatusLabel4">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="helpBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
